### ISSUE TITLE
Add support for custom language in `.vscode/settings.json`

### ISSUE THREAD

**Issue by mhkarimi1383** [file-icons/vscode#87]
I have set
```json
{
  "files.associations": {
    "*.yaml": "ansible"
  },
}
```
but icon for that is `yaml` icon

**Comment by yoyoys** [comment]
Yes this should be a missing feature, my use case is `web.nginx.conf`
<img width="303" alt="CleanShot 2023-07-12 at 11 26 03@2x" src="https://github.com/file-icons/vscode/assets/22124891/b3d136de-f6c3-40e8-a307-fd715a9e0acd">
